lib/feature_management.rb

```diff
@@ -163,8 +163,8 @@ def self.idv_by_mail_only?
       outage_status.phone_finder_outage?
   end
 
-  def self.idv_block_biometrics_requests?
-    (Identity::Hostdata.in_datacenter? && Identity::Hostdata.env == 'prod') ||
-      !IdentityConfig.store.doc_auth_selfie_capture_enabled
+  def self.idv_allow_selfie_check_in_login_prod?
+    !(Identity::Hostdata.in_datacenter? && Identity::Hostdata.env == 'prod') &&
```

Comment:
1. I agree about the method name. Updated to `idv_allow_selfie_check?` because I think that conveys the overall intent, and we will be removing the constraint on prod eventually. Do you feel strongly about the `in_current_environment` part?
2. I don't know much about this. @mitchellhenke, can you clarify? Asking you because of [this previous PR comment](https://github.com/18F/identity-idp/pull/9837#discussion_r1440717018).

Reply:
1. I don't feel strongly about the `in_current_environment?` part.  Thanks for removing the `in_login_prod?` part.

Reply:
Regarding `Identity::Hostdata.env`:

It is intended to return the name of the environment we are running in (e.g. `dev`, `int`, `staging`, `prod`, etc).

I just got on a staging host and ran it:

```ruby
irb(staging):001:0> Identity::Hostdata.env
"staging"
```

I think the confusion is that in the referenced PR comment JMax was using `Rails.env.production?`. That refers to `ENV['RAILS_ENV']` which is set to `production` in all deployed environments. That is a separate implementation from `Identity::Hostdata.env`.

ref: [method implementation](https://github.com/18F/identity-hostdata/blob/635dcf6e6c34c4277d018d75bdc02a691732c67d/lib/identity/hostdata.rb#L25)